vendor/licenses/grammar/javadoc.tmbundle.txt

```diff
@@ -1,10 +1,9 @@
 ---
 type: grammar
-name: ruby.tmbundle
+name: javadoc.tmbundle
```

Comment:
Huh!? Why's this happening? (I've not looked too deeply or played with this, just noticed it)

Reply:
I think this is two issues in one:

1. There was a `ruby.tmbundle` license in the Licenses folder which was just outdated (since we're now using the Atom Ruby package, which is not called `ruby.tmbundle`). When I ran the license checker, it removed it.

2. I added the new Javadoc grammar, which happens to have a license identical to `ruby.tmbundle` (and to most other Textmate grammars, they all share the same license).

3. Git saw a removed license, and a new added one, and marked it as a rename (h e u r i s t i c s) instead of an outdated license removed and a new one added.